packages/renderer/src/lib/ingresses-routes/IngressRouteColumnHostPath.svelte

```diff
@@ -0,0 +1,22 @@
+<script lang="ts">
+import Link from '../ui/Link.svelte';
+import type { IngressUI } from './IngressUI';
+import type { RouteUI } from './RouteUI';
+import { IngressRouteUtils } from './ingress-route-utils';
+
+export let object: IngressUI | RouteUI;
+
+const ingressRouteUtils = new IngressRouteUtils();
+</script>
+
+{#each ingressRouteUtils.getHostPaths(object) as hostPath}
+  <div class="text-sm text-gray-500">
```

Comment:
```suggestion
  <div class="text-sm text-gray-500 overflow-hidden text-ellipsis">
```

| Before | After |
| --- | --- |
| <img width="325" alt="image" src="https://github.com/containers/podman-desktop/assets/42176370/77848b84-e554-4a83-81cc-98f7de7cc5e8"> | <img width="350" alt="image" src="https://github.com/containers/podman-desktop/assets/42176370/0165508e-f58d-4df2-aed7-e8b601b462fe"> |

Reply:
Thanks @axel7083 👍